test/lib/lighthouse.js

```diff
@@ -222,7 +222,7 @@ describe('lighthouse', () => {
         return lighthouseRunner.run().then(() => {
           sandbox.assert.calledWith(reportBuilder.testSuite().testCase().className, 'www.bbc.co.uk./path/1');
           sandbox.assert.calledWith(reportBuilder.testSuite().testCase().name, '`[role]` values are valid.');
-          sandbox.assert.calledWith(reportBuilder.testSuite().testCase().time, 1200);
+          sandbox.assert.calledWith(reportBuilder.testSuite().testCase().time, 6000 / Object.keys(fakeResults.audits).length);
```

Comment:
?

Reply:
Lighthouse doesn't report a duration for each test, so it's the fake total time from the stubbed lighthouse report (6000) divided by the number of tests.

Reply:
You could use variables to clarify what you just said 😸

Reply:
Touché. Done!